backend/accounts.sql

```diff
@@ -0,0 +1,13 @@
+-- @block 
+CREATE TABLE IF NOT EXISTS account (
+    accountID INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    email VARCHAR(255) NOT NULL,
```

Comment:
Based on [Errata 1690](https://www.rfc-editor.org/errata/eid1690), email addresses are limited to 254 characters.